trunner/target/armv7a7.py

```diff
@@ -0,0 +1,92 @@
+from typing import Callable, Optional
+
+from trunner.dut import Dut, SerialDut
+from trunner.harness import (
+    HarnessBuilder,
+    PloInterface,
+    PloImageLoader,
+    PloImageProperty,
+    ShellHarness,
+    Rebooter,
+    RebooterHarness,
+    PloJffs2CleanmarkerSpec,
+)
+
+from trunner.harness import TerminalHarness
+from trunner.host import Host
+from trunner.tools import Phoenixd, Psu
+from trunner.types import TestResult, TestOptions
+from .base import TargetBase, find_port
+
+
+class ARMv7A7TargetRebooter(Rebooter):
+    def _set_flash_mode(self, flash):
+        self.host.set_flash_mode(not flash)
+
+
+class PsuPloLoader(TerminalHarness, PloInterface):
+    def __init__(self, dut: Dut, psu: Psu):
+        TerminalHarness.__init__(self)
+        PloInterface.__init__(self, dut)
+        self.psu = psu
+
+    def __call__(self):
+        """Loads plo image to RAM using psu tool and erases an area intended for rootfs."""
+        self.psu.run()
+        self.wait_prompt()
+
+        # "device id and cleanmarkers arguments are set based on target configuration in _targets/_projects"
+        self.jffs2("2.0", True, PloJffs2CleanmarkerSpec("0x10", "0x1f0", "0x10000", "0x10"), 90)
```

Comment:
please note that `PloJffs2CleanmarkerSpec` properties are of `int` type, not `string` type.

Also: device id ant cleanmarker spec can't be const here (as the comment describes they depend on particular project) - probably you should move them to `IMX6ULLEvkTarget`)

Nitpicking: I would use named parameters syntax for better readability, eg.

```python
PloJffs2CleanmarkerSpec(
    start_block=0x10,
    number_of_blocks=0x1f0
    block_size=0x10000
    cleanmarker_size=0x10
)
```

Reply:
Yeah, we missed it, thanks for your attention.

Reply:
I proposed the solution that will be pushed in a moment and I am conscious it's still not perfect; we have 2 classes `PloPsuLoader`: for armv7m7 and armv7a7, and this for armv7a7 takes jffs2 arguments.

I would like to make it common. We can move erasing part to PloImageLoader (it seems be more suitable as erasing is a part of copying a system image) and then PsuPloLoader could be declared once in base.py or even newly created common.py (?)

Reply:
However, I'd do it in the separate PR with switching from erase to jffs2 command for zynq7000.